### PR TITLE
[Keyboard] Add primer16/primer79

### PR DESCRIPTION
**Pull request by MakotoKurauchi**
## Description

add keyboard for primer16/primer79

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/yushakobo/primer16/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            key.col = 2;
+        } else {
+            key.row = 6;
+            key.col = 2;
+        }
+    }
+    action_exec((keyevent_t){.key = key, .pressed = true, .time = (timer_read() | 1)});
+    action_exec((keyevent_t){.key = key, .pressed = false, .time = (timer_read() | 1)});
+    return true;
+}
```
Comment (zvecr) [inline keyboards/yushakobo/primer16/keymaps/via/keymap.c:138]: This method is no longer being accepted into the repo, and https://docs.qmk.fm/#/feature_encoders?id=encoder-map should be used instead.

Comment (Salicylic-acid3) [reply keyboards/yushakobo/primer16/keymaps/via/keymap.c:138]: I am using this write method to remap rotary encoder rotations in VIA.
Is it possible to do the same with this ENCODER_MAP_ENABLE?
If not possible, I would prefer to leave it as is.

Comment (drashna) [reply keyboards/yushakobo/primer16/keymaps/via/keymap.c:138]: https://github.com/qmk/qmk_firmware/pull/17734

File: keyboards/yushakobo/primer16/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Primer16
+
+![Primer16](https://github.com/yushakobo/build-documents/blob/master/Primer16/imgs/Completed_assembly_Front.JPG)
```
Comment (drashna) [inline keyboards/yushakobo/primer16/readme.md:3]: Please rehost on Imgur.

File: keyboards/yushakobo/primer16/rules.mk [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+#
+BOOTMAGIC_ENABLE = yes     # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/yushakobo/primer16/rules.mk:18]: Please remove the `SLEEP_LED_ENABLE` setting, as it's not really used, and remove the comments for both it and NKRO.  
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

File: keyboards/yushakobo/primer79/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Primer79
+
+![Primer79](https://github.com/yushakobo/build-documents/blob/master/Primer79/imgs/Completed_assembly_Front.JPG)
```
Comment (drashna) [inline keyboards/yushakobo/primer79/readme.md:3]: Please host on imgur.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/yushakobo/primer16/config.h [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+#define ENCODER_RESOLUTION 4
+
+/* ws2812 RGB LED */
+#define RGB_DI_PIN D4
+
+#ifndef RGBLED_NUM
+  #define RGBLED_NUM 1
+#endif
+
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/yushakobo/primer16/config.h:45]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/yushakobo/primer16/readme.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# Primer16
+
+![Primer16](https://i.imgur.com/60cc5Nm.jpg)
```
Comment (drashna) [inline keyboards/yushakobo/primer16/readme.md:3]:
```suggestion
![Primer16](https://i.imgur.com/60cc5Nmh.jpg)
```

File: keyboards/yushakobo/primer79/readme.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# Primer79
+
+![Primer79](https://i.imgur.com/n57Td4T.jpg)
```
Comment (drashna) [inline keyboards/yushakobo/primer79/readme.md:3]:
```suggestion
![Primer79](https://i.imgur.com/n57Td4Th.jpg)
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/yushakobo/primer79/readme.md (hunk 7 above)
Comment (adeonhy) [reply keyboards/yushakobo/primer79/readme.md]: I have corrected the areas you pointed out and would appreciate your review.

File: keyboards/yushakobo/primer16/info.json [hunk 8]
```diff
@@ -0,0 +1,34 @@
+{
+    "keyboard_name": "primer16", 
+    "url": "https://shop.yushakobo.jp/products/quick17",
+    "maintainer": "yushakobo",
```
Comment (drashna) [inline keyboards/yushakobo/primer16/info.json:4]:
```suggestion
    "manufacturer": "yushakobo",
    "maintainer": "yushakobo",
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/yushakobo/primer79/info.json [hunk 9]
```diff
@@ -0,0 +1,97 @@
+{
+    "keyboard_name": "primer79", 
+    "url": "https://shop.yushakobo.jp/products/quick17",
+    "maintainer": "yushakobo",
```
Comment (drashna) [inline keyboards/yushakobo/primer79/info.json:4]:
```suggestion
    "manufacturer": "yushakobo",
    "maintainer": "yushakobo",
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**